joomla/joomla-cms
# Add sr-only behavior for active menu item.

Pull Request for Issue # .

### Summary of Changes
adds `<span class="sr-only">(current)</span></a>`
to active menu item.

### Testing Instructions
view source for active menu item.
apply patch
span should be added after link item.

### Expected result

![image](https://user-images.githubusercontent.com/1850089/64675898-e0211680-d439-11e9-9ead-f115d11fab9e.png)

### Actual result
![image](https://user-images.githubusercontent.com/1850089/64675931-f7600400-d439-11e9-936d-76e6adcc21c7.png)

### Documentation Changes Required

None
